user_guide/intro_nav_shortcuts.rst

```diff
@@ -0,0 +1,49 @@
+.. _user-guide-getting-started-shortcuts:
+
+Shortcuts
+=========
+
+Shortcuts are used to perform common actions quickly from anywhere. They can be, particularly, used to create new 
+records and navigate through the system quickly. Shortcuts are at hand, if you are not sure where in the menu the entity 
+you’re looking for is.
```

Comment:
Edit: Shortcuts are used to quickly perform common actions from anywhere. They can also be used to create new records and to navigate through the system with ease. Shortcuts are particularly useful when you are not sure where an entity is in a menu.